Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml

```diff
@@ -21,6 +21,9 @@
       icon: Interface/Actions/internal0.png
       iconOn: Interface/Actions/internal1.png
       event: !type:ToggleActionEvent
+  - type: Explosive
+    explosionType: Default
+    maxIntensity: 20
```

Comment:
20 seems a little low for gas tanks, considering these are how TTVs/tankbombs are made. hard to tell without testing in game so ill just do that later

Reply:
Remember that this is the max per tile (i.e., max of 20*15 damage per tile).
Also FWIW, I don't think there is any way to even trigger this atm. While you can manage to get above 5MPa in a canister using heaters & freezers, afaik you need to use VV to increase the release pressure to actually over pressurize a gas tank

Reply:
It's triggerable without VV I think.

Reply:
yeah you can make singletank bombs in game

Reply:
Well I have no idea how to even set them up so uhhh.. for now I'll leave as is and someone else can rebalance later.